rust/automerge-c/src/utils/string.c

```diff
@@ -0,0 +1,51 @@
+#include <stdlib.h>
+#include <string.h>
+
+#include <automerge-c/utils/string.h>
+
+int AMstrcmp(AMbyteSpan const lhs, AMbyteSpan const rhs) {
+    if (lhs.count < rhs.count) {
```

Comment:
I noticed this just because I was clicking around in the C files, but I think this incorrectly sorts "z" before "aa".